src/convertToObject.js

```diff
@@ -7,6 +7,29 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const regex = /\n+|\t+/g;
+
+  sourceString
+    .replaceAll(regex, ' ')
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item !== '')
+    .map((item) => {
+      return item.split(':').map((x) => {
+        return x
+          .split(',')
+          .map((y, index) => {
+            return index === 0 ? y.trim() : `          ${y.trim()}`;
+          })
+          .join(',\n');
```

Comment:
This block of code is attempting to handle multiple values for a single property by joining them with a comma and newline, which is not required for this task. The task is to convert CSS declarations into an object, so you should directly split the property and value and trim them without adding newlines or handling multiple values differently.